internal/commands/list.go

```diff
@@ -23,13 +24,24 @@ func List(c *cli.Context) error {
 
 	w := writer.NewTSVWriter()
 
-	header := []string{"ID", "Task", "RemindTime", "reminder"}
+	header := []string{"ID", "Task", "RemindTime", "reminder", "Priority"}
 	if err := w.Write(header); err != nil {
 		return err
 	}
 
-	for _, t := range h.GetTasks() {
-		if err := w.Write([]string{strconv.Itoa(t.ID()), t.Task(), string(t.RemindTime()), string(t.Reminder())}); err != nil {
+	tasks := h.GetTasks()
+	sort.SliceStable(tasks, func(i, j int) bool {
+		switch c.String("order") {
+		case "priority":
+			return tasks[i].Priority() < tasks[j].Priority()
+		case "id":
+			return tasks[i].ID() < tasks[j].ID()
```

Comment:
[imo]
I think sorting by `id` is not necessary. `id` is just a user interface, it may be changed every manipulation.
When user commits tasks, `task_handler` always overwrite `id` to serial number in `task_handler.align()` method. Sorting by `priority` should be run before or in `align()` method, and tasks should always be sorted by `priority`, because we only care about `priority` order.

Reply:
I made sort called in `align()` in the beginning @ 8a518af